<launch> 

  <node pkg="move_excavator"
        name="find_excavator"
        type="find_excavator"
        output="screen">
  </node>

</launch>
